modin/config/pubsub.py

```diff
@@ -35,9 +35,9 @@ class TypeDescriptor(typing.NamedTuple):
         Class description string.
     """
 
-    decode: typing.Callable[[str], object]
-    normalize: typing.Callable[[object], object]
-    verify: typing.Callable[[object], bool]
+    decode: Callable[[str], object]
+    normalize: Callable[[object], object]
```

Comment:
`Any` would disregard the return type of the function. `object` is somewhat more specific, so wouldn't that be better?

Reply:
I wonder if there's any real difference, as in some cases `decode` can e.g. return a `bool` - is that considered an `object`?..

Reply:
There is a difference because `object` is a real supertype where as using `Any` will allow any kind of type mess. Returning a `bool` is considered an `object`.